src/main/kotlin/dev/merge/client/shared/ApiClient.kt

```diff
@@ -133,8 +133,7 @@ open class ApiClient(
     }
 
     protected suspend fun <T: Any?> jsonRequest(requestConfig: RequestConfig<T>, body: Any? = null, authNames: kotlin.collections.List<String>): HttpResponse {
-        val contentType = (requestConfig.headers[HttpHeaders.ContentType]?.let { ContentType.parse(it) }
-                ?: ContentType.Application.Json)
+        requestConfig.headers[HttpHeaders.ContentType] = requestConfig.headers[HttpHeaders.ContentType] ?: "application/json"
```

Comment:
You could use `ContentType.Application.Json.contentType` as the default here as well.
```kotlin
        requestConfig.headers[HttpHeaders.ContentType] = requestConfig.headers[HttpHeaders.ContentType] ?: ContentType.Application.Json.contentType
```
But also, aren't you really just trying to set a default value if it's not present?

```suggestion
       requestConfig.headers.putIfAbsent(HttpHeaders.ContentType, ContentType.Application.Json.contentType)
```

Reply:
Thanks, the `putIfAbsent` helper is really nice! I was hesitating on using ContentType earlier due to an unrelated issue but yeah i should use the shared constant. good suggestions